core/test/financial-templates/Liquidatable.js

```diff
@@ -1363,27 +1368,30 @@ contract("Liquidatable", function(accounts) {
       // Update the liquidatableParameters to use the new token as collateral and deploy a new Liquidatable contract
       let USDCLiquidatableParameters = liquidatableParameters;
       USDCLiquidatableParameters.collateralAddress = collateralToken.address;
-      USDCLiquidationContract = await Liquidatable.new(USDCLiquidatableParameters, {
```

Comment:
+1, unrelated to PR but good cleanup.

Reply:
Yeah, it was way easier to build a function that updates the time in the store and the liqudiatable contract if the liquidatable contract was always called the same thing.